@here/harp-lines/test/rendering/RenderLines.ts

```diff
@@ -221,7 +222,7 @@ describe("Rendering lines: ", function() {
                 canvas.height / 2.0,
                 -canvas.height / 2.0
             );
-            orthoCamera.position.z = 1.0;
+            orthoCamera.position.z = 11.0;
```

Comment:
What is the reason for moving camera 10 units in Z and moving lines the same distance? It should does not change anything especially in ortho projection...

Reply:
it should not change anything but it actually did (due to the bug that is now fixed).

Reply:
So if the bug was fixed could we stay with original values? I am afraid that offsetting this entire scene just hides some stuff (bug)...